.github/workflows/linters.yml

```diff
@@ -6,21 +6,6 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  eslint:
-    name: ESLint
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup ESLint
-        run: |
-          npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x @babel/preset-env@7.x  @babel/preset-react@7.x
-          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.eslintrc.json
-          [ -f .babelrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.babelrc
-      - name: ESLint Report
-        run: npx eslint .
```

Comment:
- [ ] Why did you change the linters configuration Kindly make sure you have included the proper configuration(react-redux)

![image](https://user-images.githubusercontent.com/65068771/155747541-f4a19849-2b61-4616-b0a7-49cfa9c8527a.png)

Reply:
@andersonlebon Hi,

I did this because i contacted someone from Microverse and they said they have updated the linters for redux and react, so it's fairly to pass - but if I run them locally they are working.

Reply:
Hey @andersonlebon 

Like I mentioned, the linters have been updated and there is nothing I can on my side.

Reply:
Hey @JEFFKAY16 try to use this [configuration](https://github.com/andersonlebon/BookStore/blob/development/.github/workflows/linters.yml)